test/BRSRC13/test_resource.c

```diff
@@ -62,17 +62,21 @@ void test_resource_BrResFree_Child() {
     header = (resource_header*)(((char*)child) - sizeof(resource_header));
     TEST_ASSERT_EQUAL_INT(0xDEADBEEF, header->magic_num);
     BrResFree(r);
+
+#ifndef _DEBUG
```

Comment:
why do we need to add this guard?

Reply:
MSVC in debug mode overwrites all free'd memory with some mask to detect use-after-free.
`BrResFree(r)` will also free all its children, so we cannot access the `header` pointer anymore.

Reply:
ah, got it